Scripts/Editor/NodeEditor.cs

```diff
@@ -104,6 +104,11 @@ public class NodeEditor : XNodeEditor.Internal.NodeEditorBase<NodeEditor, NodeEd
             return NodeEditorResources.styles.nodeHighlight;
         }
 
+        /// <summary> Override to display custom node header tooltips </summary>
+        public virtual string GetHeaderTooltip(XNode.Node node) {
+            return null;
+        }
+

```

Comment:
Now that it's in NodeEditor, there is no reason to pass the node as we already have it as `target`.
Please remove the node target parameter

Reply:
You’re right, I didn’t think much about it when move the code